pkg/config/defaults.go

```diff
@@ -20,6 +20,7 @@ func setDefaults(local bool) {
 		viper.SetDefault("database.type", "local")
 		viper.SetDefault("auth.encrypt.secret_key", "THIS_MUST_BE_CHANGED_IN_PRODUCTION") // #nosec
 		viper.SetDefault(BlockstoreTypeKey, "local")
+		viper.SetDefault("blockstore.default_namespace_prefix", "local://")
```

Comment:
If it was set as part of the adapter level I would approve it in a second.
My concern is that when we use `--default-settings` it also enables us to override all the rest.
It means I can use local kv store + s3 if I'll set the other settings.
But having the default_namespace_prefix in this level - it will not go hand in hand when I'll override the default block adapter.

Reply:
Agreed, I will think it over

Reply:
suggest to deprecate this one (use it as a default) and have one in adapter level.